dispatches/models/renewables_case/wind_power.py

```diff
@@ -112,34 +111,35 @@ def build(self):
         self.electricity_out = Port(noruleinit=True, doc="A port for electricity flow")
         self.electricity_out.add(self.electricity, "electricity")
 
-        self.wind_simulation = None
+        @self.Constraint(self.flowsheet().config.time)
+        def elec_from_capacity_factor(b, t):
+            return b.electricity[t] == self.system_capacity * self.capacity_factor[t]
+
         self.setup_atb_turbine()
```

Comment:
I think, line 118 `self.setup_atb_turbine()` can be deleted. The method is called in line 126, so we can avoid the repetition.

Reply:
done